cmd/action/internal/update/update_test.go

```diff
@@ -0,0 +1,81 @@
+package update
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckForUpdate(t *testing.T) {
+	url := "https://github.com/ariga/atlas/releases/tag/test"
+	tests := []struct {
+		name           string
+		currentVersion string
+		latestVersion  string
+		store          *LatestRelease
+		checkedAt      time.Time
+		latestReleaseF func() (LatestRelease, error)
```

Comment:
Any reason this is a func? Use the value for defining the test and of you need instantiate a func in the test run body

Reply:
changed